common/teacmd/git_test.go

```diff
@@ -72,3 +73,96 @@ func cleanUpDir(targetDir string) {
 		}
 	}
 }
+
+func TestAppendToToml(t *testing.T) {
+	// Create a temporary TOML file for testing
+	tempFile, err := os.CreateTemp("", "test.toml")
+	if err != nil {
+		t.Fatalf("failed to create temporary file: %v", err)
+	}
+	defer os.Remove(tempFile.Name())
```

Comment:
you can also use `t.Cleanup`, which might be safer in cases where we use t.Parallel() for parallelizing our tests.

Reply:
updated using `t.Cleanup`